server/controllers/finance/fiscal.js

```diff
@@ -643,3 +644,28 @@ function getPeriodCurrent(date) {
 
   return db.exec(sql, [date, date]);
 }
+
+/**
+ * @method getPeriodFiscalYear
```

Comment:
@method getPeriodFiscalYear is somewhat different because the results of this function return the information for the different periods of the fiscal years except for period 0 and 13 and these data are used in the bhReportPeriodSelect component